tests/Unit/PointwiseFunctions/GeneralRelativity/Python/Test_Bindings.py

```diff
@@ -3,14 +3,28 @@
 
 from spectre.PointwiseFunctions.GeneralRelativity import *
 from spectre.DataStructures import DataVector
-from spectre.DataStructures.Tensor import tnsr
+from spectre.DataStructures.Tensor import tnsr, Scalar
 
 import numpy as np
 import numpy.testing as npt
 import unittest
 
 
 class TestBindings(unittest.TestCase):
+    def test_extrinsic_curvature(self):
+        lapse = Scalar[DataVector](num_points=1, fill=1.)
+        shift = tnsr.I[DataVector, 3](num_points=1, fill=0.)
+        deriv_shift = tnsr.iJ[DataVector, 3](num_points=1, fill=0.)
+        spatial_metric = tnsr.ii[DataVector, 3](num_points=1, fill=0.)
+        spatial_metric[0] = spatial_metric[3] = \
```

Comment:
There isn't really a use case in Python for `\` anymore. Just wrap expressions in parentheses if they need to span multiple lines, or in this case just put your expression on separate lines. Please remove the backslashes in this file.